src/routes/popup/agents/ao-yield/create-agent.tsx

```diff
@@ -0,0 +1,292 @@
+        <Content>
+          <Flex direction="column" gap={16} padding="16px 8px">
+            <Flex direction="column" gap={8} align="center" justify="center">
+              <Flex direction="row" gap={8} align="center">
+                <Text noMargin>{browser.i18n.getMessage("percentage_of_daily_ao_to_sell")}</Text>
+                <Tooltip
+                  underline
+                  content={
+                    <Text size="sm" weight="medium" style={{ maxWidth: "180px" }} noMargin>
+                      {browser.i18n.getMessage("percentage_of_daily_ao_to_sell_description")}
+                    </Text>
+                  }
+                  position="bottomEnd">
+                  <HelpCircle height={16} width={16} color={theme.tertiaryText} />
+                </Tooltip>
+              </Flex>
+              <Text size="4xl" weight="medium" noMargin>
+                {percentage}%
+              </Text>
+            </Flex>
+            <Slider value={percentage} onChange={setPercentage} min={0} max={100} minLabel="0" maxLabel="100" />
+          </Flex>
+          <Flex direction="column" gap={16}>
+            <InputButton
+              style={{ background: theme.surfaceTertiary }}
+              label={browser.i18n.getMessage("buy_asset")}
+              onClick={openAssetSelector}
+              disabled={!selectedAsset}
+              body={
+                <Flex direction="row" gap={8} align="center">
+                  <img src={selectedAsset.logo} height={24} width={24} />
+                  <Text size="lg" weight="medium" noMargin>
+                    {selectedAsset.ticker}
+                  </Text>
+                </Flex>
+              }
+              icon={
+                <Flex align="center" justify="center">
+                  <ChevronDown onClick={openAssetSelector} />
+                </Flex>
+              }
+              outerLabel
+            />
+            <Flex direction="column" gap={8}>
+              <Text noMargin>{browser.i18n.getMessage("running_time")}</Text>
+              <Flex direction="row" gap={8} align="center">
+                <InputButton
+                  style={{ background: theme.surfaceTertiary }}
+                  body={
+                    <Text size="lg" weight="medium" noMargin>
+                      {formatDate(startDate, "Start date")}
+                    </Text>
+                  }
+                  onClick={openDateSelector}
+                  outerLabel
+                />
+                <Text variant="secondary" size="base" weight="semibold" noMargin>
+                  -
+                </Text>
+                <InputButton
+                  style={{ background: theme.surfaceTertiary }}
+                  body={
+                    <Text size="lg" weight="medium" noMargin>
+                      {runIndefinitely ? "∞" : formatDate(endDate, "End date")}
+                    </Text>
+                  }
+                  onClick={openDateSelector}
```

Comment:
When clicking the end date button, the date selector should open with the end date input selected.

Also, if run indefinitely is checked, the end date button should be disabled and not open the selector when clicked.